### ISSUE TITLE
Ignoring -terraform-bin parameter when running with -output=commands

### ISSUE THREAD

**Issue by p1gmale0n** [busser/tfautomv#70]
When executing the command with the parameter -output=commands, it appears that the -terraform-bin parameter is being ignored.

example:
```
> tfautomv -terraform-bin=terragrunt --dry-run
Running "terraform init"...
Running "terraform plan"...
╷
│ Moves
│ ╷
│ │ From: random_pet.dog
│ │ To:   random_pet.cat
│ ╵
╵
> tfautomv -terraform-bin=terragrunt -output=commands
Running "terraform init"...
Running "terraform plan"...
terraform state mv "random_pet.dog" "random_pet.cat"
```

```
> tfautomv --version
v0.5.4
```

**Comment by busser** [comment]
This isn't something I anticipated, and I agree with you that what the `-output=commands` flag should respect the `-terraform-bin` flag. Thanks for reporting it!

**Comment by strowi** [comment]
Thx, was banging my head wondering why it didn't work (just starting to test tfautomv). Just stumbled into the same problem, `tfautomv --terraform-bin=terragrunt --output=commands --skip-init` shows only the moves, not the commands.

**Comment by busser** [comment]
This is fixed in [v0.6.2](https://github.com/busser/tfautomv/releases/tag/v0.6.2). Thanks again for reporting this bug!
